src/sdk/analytics/platform/vtex/index.ts

```diff
@@ -0,0 +1,48 @@
+import type { AnalyticsEvent } from '@faststore/sdk'
+
+import { send } from './search'
+
+const ONE_MINUTE_MS = 60 * 1e3
+
+setInterval(() => send({ type: 'session.ping' }), ONE_MINUTE_MS)
+
+const isFullTextSearch = (url: URL) =>
+  typeof url.searchParams.get('q') === 'string' &&
+  /^\/s(\/)?$/g.test(url.pathname)
+
+export default function sendEvent(event: AnalyticsEvent) {
+  // VTEX RC
+  window?.sendrc?.(event.name, event.params)
+
+  // VTEX Intelligent Search
+  if (event.name === 'search') {
+    send({
+      type: 'search.query',
+      text: event.params.search_term,
+      match: event.params.records_filtered,
+      url: event.params.url,
+      operator: 'and',
```

Comment:
`operator` is metadata that gets sent back on each `product_search` response. Is it possible to grab the returning value and set it correctly here?

Example below:
https://search.biggylabs.com.br/search-api/v1/storecomponents/api/split/product_search/?q=black%20top%20shirt&c=1

```
  "operator": "or",
```

There's also an unmeasured (we can run some tests later) performance increase when you keep metadata information between page changes and facets selection, such as `operator` and `fuzzy`, but we can talk about it on another occasion!

Reply:
Yeah, we don't really have access to these values in here. Also, since we don't add these as parameters to the search, I think it's ok to set them as the default used on search. If you prefer, we can remove these operator and fuzzy all together.

Reply:
Hey @tlgimenes, sorry for the delay. I had a look on our code and what would mean for some of the fields that are missing in this call. At the first moment, `operator` would be the lesser of two evils if it's set to `and` as the default (but it would impact some current and future models we're developing for synonyms).

`misspelled`, is the one that should not be missing and sending them all as `false` could have some impact in the quality of the Top Searches and Autocomplete Suggestions.

So if you could explain why that information that is coming from the API response is not available here, so that we get a clearer picture, it would be good for our internal discussion about this issue.

Thanks for the time! 💪

Reply:
Just had a meeting with the team, and how it's structured today, not having this analytics data sent back to our models will deteriorate the experience compared to other clients that currently send this data to us (other headless accounts, Store Framework, etc). There's no action that we can take at the moment to bypass this.

So if we want the best experience right now, we would need to have this data. We weren't aware of the schema.org limitation. From what I gathered, the response schema is the one used here: [ProductGalleryQuery](https://github.com/vtex-sites/base.store/blob/a9d42e413257b64718de7aab62c3a5614956c791/%40generated/graphql/persisted.json#L4
)?

cc @aprando

Reply:
> So if we want the best experience right now, we would need to have this data. We weren't aware of the schema.org limitation. From what I gathered, the response schema is the one used here: [ProductGalleryQuery](https://github.com/vtex-sites/base.store/blob/a9d42e413257b64718de7aab62c3a5614956c791/%40generated/graphql/persisted.json#L4)?

Yes, that's the one. What do you think on releasing this version, and iterating later on? We can try to measure how things goes and if it's really necessary I can try creating an middleware that adds this kind of info for you. 
The middleware could re-do the search and add these values for instance.

Reply:
Hi guys!

@tlgimenes unfortunately, as @chrsmutti  commented, if we go up without this information the system will be degraded. It's a fact, we don't even need to deploy a version to measure it.

The main problem is the misspelled attribute. Without it all terms will be considered including misspelled words and even bad words. Are we going to accept the risk of having a misspelled word or having a bad word on top searches or autocomplete?

I also don't think it's a viable option to re-do the query in a middleware. The infrastructure cost for VTEX would be very high. We should use the return of the search performed to get the info. Is the effort for this too big? Maybe it's worth a meeting to discuss and if that's the case our team can help with the implementation.

Reply:
Yes! Please don't do it! 🙏

Reply:
Ok, so I've removed this bit of the code, let's start with the clicks events first